### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by jshor** [pmndrs/drei#139]
When running Jest using create-react-app and TypeScript, I get the following error:

```
import { LineSegmentsGeometry } from "../lines/LineSegmentsGeometry.js";
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

It seems that drei is importing modules from `examples/jsm` in three, but that comes with the drawback that it cannot be properly transpiled by ts-jest

**Comment by jshor** [comment]
Closing this as it's due to the way three bundles/exports its dependencies and not due to drei itself.

For now, I've set Jest to make an exception to transpile three using the following Jest config:

```
transformIgnorePatterns: [
  'node_modules/(?!three)'
]
```

Another alternative is to set `compilerOptions.module` to `commonjs` in tsconfig.json to allow the `import` statement to work. I am using `create-react-app` for bundling however, which overwrites this option to `esnext` so I didn't have this option.
